bitwarden_license/src/Commercial.Infrastructure.EntityFramework/SecretsManager/Repositories/ProjectRepository.cs

```diff
@@ -103,4 +103,21 @@ public async Task<bool> UserHasWriteAccessToProject(Guid id, Guid userId)
 
         return await query.AnyAsync();
     }
+
+    public async Task<IEnumerable<Core.SecretsManager.Entities.Project>> ImportAsync(IEnumerable<Core.SecretsManager.Entities.Project> projects)
+    {
+        using (var scope = ServiceScopeFactory.CreateScope())
+        {
+            var entities = new List<Project>();
+            foreach (var p in projects)
+            {
+                var entity = Mapper.Map<Project>(p);
+                entities.Add(entity);
+            }
```

Comment:
We can use Linq here to build our new list.

```suggestion
            var entities = projects.Select(p => Mapper.Map<Project>(p));
```

Reply:
Updated